9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I guess it used to not be possible, but it does sound great to me. Both cases should be possible and documented I think, but we may suggest the mounting solution as the better option.